db/migrate/20220325141647_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+      t.references :users, null: false, foriegn_key: true, index:true
```

Comment:
- [ ] Let's reference `author`, instead of the `user`. This will give us an `author_id` which corresponds with the ERD foreign_key column. We should do the same to all tables(migration files) that reference a `user`.

Reply:
@elisha2kyakpo1 Thank you for the review. 
Done